### PR TITLE
add autoround example

### PR DESCRIPTION
**Pull request by WeiweiZhang1**
## Type of Change

example and documentation
API not changed

### CODE REVIEW

**Comment by changwangss** [comment]
if you are urgent, I can agree first, but then we need create the auroround config and merge example to run_generation.py

File: examples/huggingface/pytorch/text-generation/quantization/auto_round/README.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+```
+- **Utilizing the AdamW Optimizer:**
+Include the flag `--adam`. Note that AdamW is less effective than Sign gradient descent in many scenarios we tested.
+
+- **Running the Original SignRound:**
+```bash
+CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1 --iters 400 --lr 0.0025 --minmax_lr 0.0025
+```
+ `--enable_minmax_tuning` is strongly recommended 
+
```
Comment (VincyZhang) [inline examples/huggingface/pytorch/text-generation/quantization/auto_round/README.md:89]: Will you add running performance/accuracy using neural_speed and lead to this page: https://github.com/intel/intel-extension-for-transformers/tree/main/examples/huggingface/neural_speed

**Review by VincyZhang — APPROVED** [review]

**Review by kevinintel — APPROVED** [review]

File: docs/imgs/W2G128.png
[the remark this answers is not in the document]
Comment (WeiweiZhang1) [reply docs/imgs/W2G128.png:1]: It will be fixed in next PR
